### ISSUE TITLE
Template switcher in full view and repeatable option for fields.

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#256]
I opened ticket on Joomcoders but will also post here so in case you miss it there.
 
As we know there is option to set multiple templates in list view of JoomCCK and users will be able to switch between list templates. I would like to have same solution in full view.

1. I am using one section as a database of CV Resume, so each registered user will be able to create their resume and I want to let them after submitting their resume They will see it in default template, but they should have option to choose other templates and see different design of Resume. 
2. In URL field we have "+ button"  which gives us option to add multiple links as value of one field. Can you modify this field? I mean create new field with same features. To be more clear My idea is to have a group of fields in one field and be able to submit multiple times. This is used to show for example education, Experience or other stuff in CV Resume. You can see in video https://youtu.be/Eu5QCIe2syQ, there are multiple different type of fields, dropdown, select, input, calendar which are saved as a single value and there is option to add more like this. You can test this form on this website https://myjobs.ge/en/cv 

Can you write price and time needed to create this two features separately? More urgent is Second feature.

**Comment by andiamine** [comment]
View switcher still working, you need to select multiple records template. and make sure icon is enabled in markup template settings
![Screenshot 2025-01-03 231003](https://github.com/user-attachments/assets/698173d4-72dc-470c-85d2-ad7819d0b736)

**Comment by Giorgi625** [comment]
Yes I know it works but it is in List view, I need same feature in Full view.

**Comment by andiamine** [comment]
yes is possible we can do as custom work.

**Comment by Giorgi625** [comment]
ok it is not urgent so I will remind you about it later.
